oct/cli/package/common_options.py

```diff
@@ -14,21 +14,29 @@ def package_options(func):
     :return: decorated CLI command
     """
     return option(
-        '--update/--upgrade',
-        '-d/-g',
-        'update_current_stage',
-        default=False,
-        help='Update stage or upgrade to next stage.  [default: upgrade]',
+        '--upgrade',
```

Comment:
Or we could use a flat list of options but that means that we'll have to reserve names
```
oct package ami --stage current
oct package ami --stage next
oct package ami --stage fork
```
I don't feel strong for any of those. @jhadvig @jupierce got any preferences?